internal/receiver/databricksreceiver/metadata.yaml

```
@@ -62,3 +94,1118 @@ metrics:
+    enabled: true
+    description: n/a
+    sum:
+      value_type: int
+      monotonic: false
+      aggregation: cumulative
+    attributes:
+      [cluster_id, spark_app_id, pipeline_id, pipeline_name]
+  databricks.spark.databricks.directorycommit.markerrefresherrors:
+    enabled: true
+    description: n/a
+    sum:
+      value_type: int
+      monotonic: false
+      aggregation: cumulative
+    attributes:
+      [cluster_id, spark_app_id, pipeline_id, pipeline_name]
+  databricks.spark.databricks.directorycommit.markersread:
+    enabled: true
+    description: n/a
+    sum:
+      value_type: int
+      monotonic: false
+      aggregation: cumulative
+    attributes:
+      [cluster_id, spark_app_id, pipeline_id, pipeline_name]
+  databricks.spark.databricks.directorycommit.repeatedlistcount:
+    enabled: true
+    description: n/a
+    sum:
+      value_type: int
+      monotonic: false
+      aggregation: cumulative
+    attributes:
+      [cluster_id, spark_app_id, pipeline_id, pipeline_name]
+  databricks.spark.databricks.directorycommit.uncommittedfilesfiltered:
+    enabled: true
+    description: n/a
+    sum:
+      value_type: int
+      monotonic: false
+      aggregation: cumulative
+    attributes:
+      [cluster_id, spark_app_id, pipeline_id, pipeline_name]
+  databricks.spark.databricks.directorycommit.untrackedfilesfound:
+    enabled: true
+    description: n/a
+    sum:
+      value_type: int
+      monotonic: false
+      aggregation: cumulative
+    attributes:
+      [cluster_id, spark_app_id, pipeline_id, pipeline_name]
+  databricks.spark.databricks.directorycommit.vacuumcount:
+    enabled: true
+    description: n/a
+    sum:
+      value_type: int
+      monotonic: false
+      aggregation: cumulative
+    attributes:
+      [cluster_id, spark_app_id, pipeline_id, pipeline_name]
+  databricks.spark.databricks.directorycommit.vacuumerrors:
+    enabled: true
+    description: n/a
+    sum:
+      value_type: int
+      monotonic: false
+      aggregation: cumulative
+    attributes:
+      [cluster_id, spark_app_id, pipeline_id, pipeline_name]
+  databricks.spark.databricks.preemption.numchecks:
+    enabled: true
+    description: n/a
+    sum:
+      value_type: int
+      monotonic: false
+      aggregation: cumulative
+    attributes:
+      [cluster_id, spark_app_id, pipeline_id, pipeline_name]
+  databricks.spark.databricks.preemption.numpoolsautoexpired:
+    enabled: true
+    description: n/a
+    sum:
+      value_type: int
+      monotonic: false
+      aggregation: cumulative
+    attributes:
+      [cluster_id, spark_app_id, pipeline_id, pipeline_name]
+  databricks.spark.databricks.preemption.numtaskspreempted:
+    enabled: true
+    description: n/a
+    sum:
+      value_type: int
+      monotonic: false
+      aggregation: cumulative
+    attributes:
+      [cluster_id, spark_app_id, pipeline_id, pipeline_name]
+  databricks.spark.databricks.preemption.poolstarvationmillis:
```

I think taking https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/README.md into consideration would make this something like:

```suggestion
  databricks.spark.databricks.preemption.poolstarvation.time:
```

with the unit metadata taking care of ms.

---

Fixed.